app/index.js

```diff
@@ -324,28 +333,23 @@ module.exports = generators.Base.extend({
       var prompts = [{
         name: 'swaggerChoice',
         type: 'list',
-        message: 'Swagger file to use:',
+        message: 'Swagger file to use to create endpoints and companion iOS SDK:',
         choices: [choices.customSwagger, choices.exampleEndpoints],
         default: []
       },{
         name: 'path',
         type: 'input',
         message: 'Provide the path to a swagger file:',
-        filter: function(response) {
-          return response.trim();
-        },
-        validate: function(response) {
-          // permit paths starting with 'filename' or '/' or './' or '../' or 'http://' or 'https://'
-          var pathPattern = new RegExp(/^\w+|^\/|^\.\.?\/|^https?:\/\/\S+/);
-          return pathPattern.test(response);
-        },
+        filter: function(response) { return response.trim(); },
+        validate: validateFilePath,
         when: function(question) {
                 return (question.swaggerChoice === choices.customSwagger);
               }
       }];
       this.prompt(prompts, function(answers) {
         if (answers.swaggerChoice === choices.exampleEndpoints) {
           this.exampleEndpoints = true;
+          this.fromSwagger = path.join(__dirname, '../refresh/templates/common/productSwagger.yaml');
```

Comment:
My impression is that this is possible as long as we generate the SDKs before anything else. I can start moving this, but might take a bit more time.